### PR TITLE
docs: row span tutorial

### PR DESCRIPTION
**Pull request by js87zz**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description
* add row span tutorial

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

File: docs/ko/row-span.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# 로우 스팬 🔩
```
Comment (jung-han) [inline docs/ko/row-span.md]: 이모지로 🗄나 🖖 어떠나요? ㅋㅋ

Comment (jung-han) [reply docs/ko/row-span.md]: `# 행 확장`
은 어떤가요?

File: docs/ko/row-span.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+`appendRow` 와 `removeRow` API에는 로우 스팬이 적용된 경우만 동작하는 옵션이 있다.
+
+| API | 옵션 | 설명 |
+| --- | --- | --- |
+| `appendRow`| `extendPrevRowSpan` | 추가된 로우의 이전(위) 로우가 병합된 셀의 가장 하단 로우인 경우, 이를 확장하여 추가 병합할 지 결정하는 옵션이다. 만약 로우가 병합된 셀의 중간에 추가되는 경우는 옵션 값과 상관없이 추가 병합된다. |
+| `removeRow`| `keepRowSpanData` | 로우 스팬이 적용된 가장 상단의 로우가 삭제된 경우(병합된 셀의 가장 상단 로우), 다음 로우를 기준으로 로우 스팬을 유지할 지 결정하는 옵션이다. 만약 병합된 셀의 중간에 있는 로우가 삭제된 경우는 옵션 값과 상관없이 기존 병합 상태를 유지한다.|
+
+## 예제
+
+[여기](http://nhn.github.io/tui.grid/latest/tutorial-example06-attributes)서 로우 스팬 예제를 확인할 수 있다.
```
Comment (jung-han) [inline docs/ko/row-span.md]: `docs/ko/readme` 목차에 추가 해줘야할 것 같습니다!

File: docs/ko/row-span.md (hunk 1 above)
Comment (js87zz) [reply docs/ko/row-span.md]: `행 확장`은 행 전체의 길이를 늘리거나 행끼리 합친다는 느낌같아서..이대로 두는게 더 의미전달에 좋을 것 같습니다!

**Pull request merged**